pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -51,6 +51,7 @@ func newPandCLifecycles(management *config.ManagementContext) (*projectLifecycle
 		nsLister:           management.Core.Namespaces("").Controller().Lister(),
 		prtbLister:         management.Management.ProjectRoleTemplateBindings("").Controller().Lister(),
 		crtbLister:         management.Management.ClusterRoleTemplateBindings("").Controller().Lister(),
+		crtbs:              management.Management.ClusterRoleTemplateBindings(""),
```

Comment:
nit: maybe name this crtbClient to conform, or don't create a field at all and take it of l.mgr.mgt

Reply:
I added this because the test was initially initially in a different package where crtbInterface was added, and I can't call it in the test from l.mgr.mgt as I can't mock mgt. 
I think the name is fine since that's the convention we follow for all controllers?